### DISCUSSION TITLE
(Resuelto) Alguien más tiene problemas al acceder a Microsoft Teams desde el domingo

### DISCUSSION THREAD

**Discussion by richirrim** [LaunchX-InnovaccionVirtual/CursoIntroPython#240] [Q&A] [answered]
(Resuelto) Este error me empezó a salir desde el domingo pero pensé que  era normal, que se había caído temporalmente. Pero hoy ya es Lunes y sigue sin dejarme acceder. Ya intente cerrar y volver a iniciar sesión como sugiere pero nada

![Captura de pantalla 2022-02-14 12 48 18](https://user-images.githubusercontent.com/11398329/153928409-363b0991-93df-4ac2-b880-bf15eae8bc75.png)
.

**Comment by doguedogue** [comment, marked as answer]
No me ha dado problemas Teams
![image](https://user-images.githubusercontent.com/23409026/153929538-0a36499d-4787-4817-aaa5-c02a1413a04c.png)

Intenta desde [office.com](https://www.office.com/) con tus credenciales de @innovaccion.com

🛰️ Ninja de performance es la persona con la que podrás contactar si tienes algún problema técnico con la plataforma, el correo es contacto@innovaccion.mx y deberás ser lo más claro y descriptivo para que te pueda ayudar

**Reply by richirrim** [reply]
Gracias por tu respuesta, parece ser que Google Chrome se volvió loco. Acabo de probar iniciar con otro navegador y me dejo acceder con normalidad.

**Comment by CesarMezaB** [comment]
Intenta acceder teams por web para ver si funciona por ahí...

**Reply by richirrim** [reply]
Gracias por tu respuesta, al perecer un bloqueador de ads, era el problema.
